org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java

```diff
@@ -103,8 +103,9 @@ public void endElement(String uri, String localName, String qName) {
 
 	private static final String[] TOCS = {
 			"org.eclipse.platform.doc.isv",
-			"org.eclipse.jdt.doc.isv",
-			"org.eclipse.pde.doc.user",
+			// FIXME why do we test JDT + PDE here ?!?
```

Comment:
The ApiDocTest that run **on jenkins** is meaningless. It is supposed to run in full SDK build and test that all the API we ship in SDK doesn't have broken references - and that needs both JDT / PDE.

So if you want fix jenkins build, please **conditionally** disable it for jenkins.